README.md

```diff
@@ -640,7 +645,7 @@ const observers = await arIO.getPrescribedObservers();
   {
     "gatewayAddress": "BpQlyhREz4lNGS-y3rSS1WxADfxPpAuing9Lgfdrj2U",
     "observerAddress": "2Fk8lCmDegPg6jjprl57-UCpKmNgYiKwyhkU4vMNDnE",
-    "stake": 10000,
+    "stake": 10000, // value in mIO
```

Comment:
Should maybe update this readme example as I think this wouldn't be a valid amount of mIO for operatorStake